apps/freelance/src/pages/ExampleRootPage/index.tsx

```diff
@@ -2,6 +2,7 @@ import { Link } from 'react-router-dom';
 import { ExampleButton } from '@freelance/components';
 import { Wrapper, Title, Text } from './styles';
 
+// eslint-disable-next-line
```

Comment:
hmm it does not make sense create the rule and then disable it

Reply:
fixed. Mykhailo have sent you PR with fixes. Will add upon approval